### PR TITLE
handbook eng: simulate k8s admin security restrictions

### PR DESCRIPTION
**Pull request by uwedeportivo**
describe how to simulate managing a sourcegraph instance in k8s with restricted admin powers subject to pod security policies

### CODE REVIEW

File: handbook/engineering/distribution/k8s_admin_custom_policy.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+```
+
+First we need a pod security policy that matches as close as possible the restrictions at the customer site. For example
+(this is just an example, tweak as necessary):
+
+```yaml
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: foopolicy
```
Comment (nicksnyder) [inline handbook/engineering/distribution/k8s_admin_custom_policy.md]: Can we avoid using `foo` here an elsewhere? It makes it seem less realistic. If you were actually going to configure a policy for our cluster, what would you call it?

Comment (uwedeportivo) [reply handbook/engineering/distribution/k8s_admin_custom_policy.md]: fixed

**Comment by uwedeportivo** [comment]
> Is this something that only distribution teammates will need to do?

Probably. Distribution needs to test that our configs work without requiring unreasonable privileges.

**Comment by uwedeportivo** [comment]
i'm going to update this with namespaces before merging

**Pull request merged**